tests/certification/pubsub/azure/eventhubs/delete-eventhub.sh

```diff
@@ -0,0 +1,38 @@
+# ------------------------------------------------------------
+# Copyright 2021 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+
+set -e
+
+# This script delete the Azure Eventhub created in the scoped Azure EventHub namespace as per the test suite.
+# Step 1 : Login to Azure with provided SPN which has `Azure Event Hubs Data Owner` role on scoped eventhub namespace.
+# Step 2 : Delete the eventhub/topic which is provided as an argument
```

Comment:
Deleting the checkpoint container related to this eventhub and consumer will add to the reliability of the test.

Reply:
taken care.